testsuite/features/step_definitions/command_steps.rb

```diff
@@ -879,6 +879,10 @@
     cmd = "apt-get --assume-yes install #{package}"
     successcodes = [0]
     not_found_msg = 'Unable to locate package'
+  elsif host.include? 'slemicro'
+    cmd = "transactional-update pkg install -n #{package}"
+    successcodes = [0, 100, 101, 102, 103, 106]
```

Comment:
Are we sure the success codes are the same for this different command?

Reply:
No. I`ll check what the man page says about that.

Reply:
https://github.com/openSUSE/transactional-update/blob/master/man/transactional-update.8.xml was not very helpful.

Reply:
```bash
micro-min-slemicro52:~ # transactional-update pkg install vim
Checking for newer version.
transactional-update 4.0.0~rc2 started
Options: pkg install vim
Separate /var detected.
2023-01-27 13:57:00 tukit 4.0.0~rc2 started
2023-01-27 13:57:00 Options: -c1 open 
2023-01-27 13:57:00 Using snapshot 1 as base for new snapshot 2.
2023-01-27 13:57:00 No previous snapshot to sync with - skipping
ID: 2
2023-01-27 13:57:00 Transaction completed.
Calling zypper install
2023-01-27 13:57:00 tukit 4.0.0~rc2 started
2023-01-27 13:57:00 Options: callext 2 zypper -R {} install vim 
2023-01-27 13:57:00 Executing `zypper -R /.snapshots/2/snapshot install vim`:
Loading repository data...
Reading installed packages...
'vim' not found in package names. Trying capabilities.
No provider of 'vim' found.
Resolving package dependencies...
Nothing to do.
2023-01-27 13:57:00 Application returned with exit status 104.
ERROR: zypper install on /.snapshots/2/snapshot failed with exit code 104!
Removing snapshot #2...
2023-01-27 13:57:00 tukit 4.0.0~rc2 started
2023-01-27 13:57:00 Options: abort 2 
2023-01-27 13:57:00 Discarding snapshot 2.
2023-01-27 13:57:01 Transaction completed.
transactional-update finished
micro-min-slemicro52:~ # echo $?
1
```
The command is using zypper in the background but I guess we see the ususal `0, 1` return codes here?

Reply:
I only see `0, 1` as return codes. Nevertheless I would leave the code as is for the moment.